kontrakt/src/main/java/no/nav/abakus/iaygrunnlag/oppgittopptjening/v1/OppgittYtelseDto.java

```diff
@@ -0,0 +1,60 @@
+package no.nav.abakus.iaygrunnlag.oppgittopptjening.v1;
+
+import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.*;
+import no.nav.abakus.iaygrunnlag.Periode;
+import no.nav.abakus.iaygrunnlag.kodeverk.ArbeidType;
+import no.nav.abakus.iaygrunnlag.kodeverk.Landkode;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(value = Include.NON_ABSENT, content = Include.ALWAYS)
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.NONE, getterVisibility = JsonAutoDetect.Visibility.NONE, setterVisibility = JsonAutoDetect.Visibility.NONE, isGetterVisibility = JsonAutoDetect.Visibility.NONE, creatorVisibility = JsonAutoDetect.Visibility.NONE)
+public class OppgittYtelseDto {
```

Comment:
Her kan vi evt. forenkle til å bruke record og færre annoteringer, ref teknisk møte for en stund siden. Eller la det være slik, for konsistens mot resten av applikasjonen

Reply:
Ja, men kanskje greit at det er likt som du seier?

Reply:
vi bør bruke records siden vi ble enige om det
https://github.com/navikt/k9-verdikjede/blob/master/docs/arkitektur/adr/dto-typer.md